app/commands/user_track/generate_concepts_status.rb

```diff
@@ -0,0 +1,27 @@
+class UserTrack
+  class GenerateConceptsStatus
```

Comment:
- I prefer pluralization as `ConceptStatuses` here as the pluralized form.
- And as we're not generating statuses, we should add something on the end. Probably `Hash`, which I don't love, but nothing better strikes me yet :)

```suggestion
  class GenerateConceptStatusesHash
```

Reply:
I edited this ^^